CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * [#6137](https://github.com/rubocop-hq/rubocop/pull/6137): Allow `db` to allowed names of `Naming/UncommunicativeMethodParamName` cop in default config. ([@mkenyon][])
 
+### Bug fixes
+
+* [#6127](https://github.com/rubocop-hq/rubocop/issues/6127): Fix an error for `Layout/ClosingParenthesisIndentation` when method arguments are empty with newlines. ([@tatsuyafw][])
```

Comment:
Would you please move the `### Bug fixes` section before the `### Changes` section?

Others looks good 👍

Reply:
Thanks!

I've moved the `Bug fixes` section to before the `Changes` section, and rebased master to resolve conflicts